assembly/src/release/bin/activemq

```diff
@@ -346,6 +346,7 @@ invokeJar(){
               --add-opens java.rmi/sun.rmi.transport.tcp=ALL-UNNAMED \
               --add-opens java.base/java.util.concurrent=ALL-UNNAMED \
               --add-opens java.base/java.util.concurrent.atomic=ALL-UNNAMED \
+              --add-opens java.base/sun.nio.ch=ALL-UNNAMED \
```

Comment:
AFAIR, this package is not available on all JDK (not sure adoptium has it for instance).